### PR TITLE
type alias documentation

### PR DESCRIPTION
**Pull request by gluax**
Type alias documentation. I'm not sure what else to add, but type aliases themselves are fairly straightforward.

### CODE REVIEW

File: documentation/developer/language/03_types.md [hunk 1]
```diff
@@ -96,6 +96,22 @@ let e: char = '🤷🏿‍♀️'; // illegal
...
+type int = u32;
+
+function add(a: int, b: int) -> int {
+    let _added: int = a + b; // This works.
+    let addedu32: u32 = a + b; // This also works since it's the same as a u32.
+    return addedu32;
+}
+```
+
+However, do note that type inference would never infer a value as an alias, as aliases themselves reduce to the standard included types.
```
Comment (acoglio) [inline documentation/developer/language/03_types.md:114]: would never infer a value as an alias
=>
never infers a type alias (e.g. for a variable)

**Review by acoglio — APPROVED** [review]
Looks good. I added a suggestion.

File: documentation/developer/language/03_types.md (hunk 1 above)
Comment (gluax) [reply documentation/developer/language/03_types.md]: will do!

**Review by damirka — APPROVED** [review]

**Comment by gluax** [comment]
@acoglio can we merge this now? It's targeted towards the next-release branch, so it should be fine to merge it!

**Pull request merged**